src/frontend/ts/state/chessState.ts

```diff
@@ -1,23 +1,29 @@
 import Urbit from '@urbit/http-api'
-import { Ship, GameID, GameInfo, ActiveGameInfo, Challenge, ChessUpdate, ChallengeUpdate } from '../types/urbitChess'
+import { Ship, GameID, SAN, FENPosition, Move, GameInfo, ActiveGameInfo, Challenge, ChessUpdate, ChallengeUpdate } from '../types/urbitChess'
 
 interface ChessState {
   urbit: Urbit | null
   displayGame: ActiveGameInfo | null
+  displayMoves: Array<SAN> | null
   practiceBoard: String | null
   activeGames: Map<GameID, ActiveGameInfo>
+  activeGameMoves: Map<GameID, Array<Move>>
   incomingChallenges: Map<Ship, Challenge>
   outgoingChallenges: Map<Ship, Challenge>
   friends: Array<Ship>
+  displayIndex: number | null
+
```

Comment:
Is this space meant to delimit members from functions?

Reply:
Yeah, but looking at it now it's a bit of a Hoonish thing to do here; I'll break it up with comments like a normal dev.